powertools-core/src/main/java/software/amazon/lambda/powertools/core/internal/UserAgentConfigurer.java

```diff
@@ -0,0 +1,90 @@
+package software.amazon.lambda.powertools.core.internal;
+
+import com.sun.org.slf4j.internal.Logger;
+import com.sun.org.slf4j.internal.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Properties;
+
+import static software.amazon.lambda.powertools.core.internal.SystemWrapper.getenv;
+
+public class UserAgentConfigurer {
```

Comment:
```suggestion
public class UserAgentConfigurator {
```

Reply:
And a bit of javadoc would be nice - this is here so that we can add extra info to the user agent in the AWS SDK sessions.

The python impl and docs are here --> 
https://github.com/aws-powertools/powertools-lambda-python/blob/021076efab85cb0afa867a0ffa68060917733f99/aws_lambda_powertools/shared/user_agent.py#L75

Reply:
Done!